### ISSUE TITLE
Material Sample App not working

### ISSUE THREAD

**Issue by alexandreseabra** [angular/material2#3147]
#### Bug, feature request, or proposal:
BUG

#### What is the expected behavior?
Show the Sample application

#### What is the current behavior?
Just freezes on "Loading document..."

#### What are the steps to reproduce?
Access: https://material.angular.io/guide/material2-app from chome on linux.

Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
Plunker template: http://plnkr.co/edit/o077B6uEiiIgkC0S06dd

#### What is the use-case or motivation for changing an existing behavior?
You guys did a great job. But if the sample code is not working, beginners get confused

#### Which versions of Angular, Material, OS, browsers are affected?
The one in https://material.angular.io/guide/material2-app.

#### Is there anything else we should know?
![screenshot from 2017-02-16 21-58-09](https://cloud.githubusercontent.com/assets/226781/23046947/d655f306-f493-11e6-9dad-906600508444.png)

**Comment by andrewseguin** [comment]
Hey @DevVersion can you check this out? It's related to #2770. The sample app link at the bottom of the getting started guide changed its link through the docs script.

**Comment by demirk4n** [comment]
@andrewseguin  First of all, thank you for your team for all the amazing work! 

The `Material Sample App` link in master `getting-started.md` points to repo of [jelbourn/material2-app](https://github.com/jelbourn/material2-app) which is unfortunately quite outdated ATM. I am not sure if that would be any help to beginners in its current outdated status.

Until it gets fixed I created a fork and maintaining it on [demirk4n/angular-material-demo](https://github.com/demirk4n/angular-material-demo) 
For that; I updated the code for mobile devices,  added new  components and simple tests with automatic dependency and build checks. 

I hope that helps. Thank you.
